build/unix/module.modulemap

```diff
@@ -14,8 +14,6 @@ module ROOT_Foundation_C  {
 module ROOT_Config  {
   // These headers are supposed to be only textually expanded for each TU.
   module "RVersion.h" { textual header "RVersion.h" export * }
-  module "RConfig.h" { header "RConfig.h" export * }
-  module "ROOT/RConfig.h" { header "ROOT/RConfig.h" export * }
```

Comment:
@vgvassilev these clash with headers from R with the same name. Do we really need these (they just fwd include to `ROOT/RConfig.hxx`) in the module?

Reply:
Do we need `RConfig.h` you mean?

Reply:
Why ? In principle no ROOT file will include RConfig.h but they will use ROOT/RConfig.hxx

Reply:
I don't think many people outside ROOT include this file. They include ROOT headers that include ROOT/RConfig.hxx. And probably the duplication is just for a forward declaration, so it should not be a big thing.
Anyway, I have reverted those changes, probably this was not the cause of the conflict seen before, It was caused by the inclusion of  Rconfig.h in G__Core.cxx, that is now fixed. 
Let's see what will be  the result of the CI builds